app/backend/src/clj/teet/contract/contract_commands.clj

```diff
@@ -153,3 +154,30 @@
               {:db/id company-contract-eid
                :company-contract/employees "new-company-contract-employee"}]})
 
+(defcommand :thk.contract/add-new-contract-employee
+  {:doc "Add a new contract employee"
+   :payload {form-value :form-value
+             company-contract-eid :company-contract-eid}
+   :context {:keys [user db]}
+   :project-id nil
+   :authorization {:contracts/contract-editing {}}}
+  (let [employee-fields (-> (select-keys form-value [:user/given-name :user/family-name
+                                                     :user/email :user/phone-number]))
+        user-person-id (user-model/normalize-person-id (:user/person-id form-value))
+        new-employee-id "new-employee"
+        tempids
+        (:tempids (tx [(merge
```

Comment:
This should probably use the `ensure-unique-email` wrapper which is used in the `admin/create-user`